src/views/MobilitySettingsView/components/RouteLength/RouteLength.js

```diff
@@ -28,6 +28,78 @@ const RouteLength = ({ classes, intl, route }) => {
             {intl.formatMessage({ id: 'mobilityPlatform.menu.bicycleRoutes.auraRiverTrail' })}
           </Typography>
         );
+      case 'Seutureitti 1':
```

Comment:
Does some crap like this work?
let text = "";
for (let i = 1; i < 13; i++) {
      return (
          <Typography component="p" variant="body2" className={classes.margin}>
            {intl.formatMessage({ id: 'mobilityPlatform.menu.bicycleRoutes.regionalTrail'+i })}
          </Typography>
        );
}

Reply:
I'll look into it.

Reply:
I refactored it to reduce repetition.